kie-api/src/build/revapi-config.json

```diff
@@ -1,41 +1,85 @@
 {
-  "filters": {
-    "revapi": {
-      "java": {
-        "filter": {
-          "classes": {
-            "regex": true,
-            "exclude": []
-          }
+    "filters": {
+        "revapi": {
+            "java": {
+                "filter": {
+                    "classes": {
+                        "regex": true,
+                        "exclude": []
+                    }
+                }
+            }
+        }
+    },
+    "ignores": {
+        "revapi": {
+            "_comment": "Changes between 7.33.0.Final and the current branch. These changes are desired and thus ignored.",
+            "ignore": [
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method java.util.List<java.lang.String> org.kie.api.event.process.ProcessVariableChangedEvent::getTags()",
+                    "package": "org.kie.api.event.process",
+                    "classSimpleName": "ProcessVariableChangedEvent",
+                    "methodName": "getTags",
+                    "elementKind": "method",
+                    "justification": "add the possibility to retrieve custom tags associated with variables"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method boolean org.kie.api.event.process.ProcessVariableChangedEvent::hasTag(java.lang.String)",
+                    "package": "org.kie.api.event.process",
+                    "classSimpleName": "ProcessVariableChangedEvent",
+                    "methodName": "hasTag",
+                    "elementKind": "method",
+                    "justification": "add the possibility to find out if custom tag is associated with variables"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method java.lang.Object org.kie.api.runtime.manager.audit.VariableInstanceLog::getValueObject()",
+                    "package": "org.kie.api.runtime.manager.audit",
+                    "classSimpleName": "VariableInstanceLog",
+                    "methodName": "getValueObject",
+                    "elementKind": "method",
+                    "justification": "necessary when full audit is neede to restore the original object"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method org.kie.api.runtime.process.ProcessInstance org.kie.api.runtime.process.ProcessRuntime::startProcessFromNodeIds(java.lang.String, java.util.Map<java.lang.String, java.lang.Object>, java.lang.String[])",
+                    "package": "org.kie.api.runtime.process",
+                    "classSimpleName": "ProcessRuntime",
+                    "methodName": "startProcessFromNodeIds",
+                    "elementKind": "method",
+                    "justification": "new functionality added"
```

Comment:
Can you please specify what this functionality is added?

Reply:
done